protocol/messenger_communities.go

```diff
@@ -2306,3 +2312,31 @@ func (m *Messenger) cleanUpFailedImport(community *communities.Community) {
 func (m *Messenger) publishImportProgress(progress *discord.ImportProgress) {
 	m.config.messengerSignalsHandler.DiscordCommunityImportProgress(progress)
 }
+
+func (m *Messenger) startPublishImportProgressInterval(c chan *discord.ImportProgress, done chan struct{}) {
+
+	var currentProgress *discord.ImportProgress
+
+	go func() {
+		ticker := time.NewTicker(1 * time.Second)
```

Comment:
I think 1 second is reasonable. If anybody thinks this should be bigger/smaller, let me know